pkg/forklift-api/services/tls_certificate.go

```diff
@@ -0,0 +1,42 @@
+package services
+
+import (
+	"encoding/pem"
+	"net/http"
+
+	"github.com/konveyor/forklift-controller/pkg/controller/provider/container"
+	core "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func serveTlsCertificate(resp http.ResponseWriter, req *http.Request, client client.Client) {
+	if url := req.URL.Query().Get("URL"); url != "" {
+		log.Info("received a request to retrieve certificate", "url", url)
+		secret := &core.Secret{
+			Data: map[string][]byte{"insecureSkipVerify": []byte("true")},
```

Comment:
why you set it to true instead of checking the secret? or we get here only if cert wasn't set - then we know it's insecure?

Reply:
this code is executed before we have a secret - it's supposed to return a TLS certificate to clients over an insecure channel